src/extension.ts

```diff
@@ -182,7 +183,32 @@ const createHover = (document: vscode.TextDocument, position: vscode.Position):
   return new vscode.Hover(hoverContent);
 };
 
+const setup = () => {
+  vscode.workspace.findFiles('**/launch.json')
+  .then(files => {
+    const setup = require(files[0].path);
+    const setupPath = path.join(__dirname, 'setup.sh');
+    let tizenStudioPath;
+    let IoTjsPath;
+    setup.configurations.forEach(i => {
+      if (i.tizenStudioPath !== undefined && i.tizenStudioPath !== null) {
+        if (i.IoTjsPath !== undefined && i.IoTjsPath !== null) {
+          tizenStudioPath = i.tizenStudioPath;
+          IoTjsPath = i.IoTjsPath;
+        }
+      }
+    });
```

Comment:
Does this mean that we will use the last configuration, where both of the `tizenStudioPath` and `IoTjsPath` were defined?
Furthermore, this can be simplified:
```
if (i.tizenStudioPath && i.IoTjsPath) {
  tizenStudioPath = i.tizenStudioPath;
  IoTjsPath = i.IoTjsPath;
}
```